.github/workflows/package-nodejs-viorn-lib-publich.yml

```diff
@@ -0,0 +1,37 @@
+name: NodeJs Viron/lib Package CI
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - "packages/nodejs/**"
+      - "package.json"
```

Comment:
is this necessary?
Code Build we are currently using does not seem to include it in filter.
<img width="1520" alt="image" src="https://github.com/cam-inc/viron/assets/62197019/3792c382-a1d7-40f8-b7eb-174958d6f591">

Reply:
- "package.json"
no needed and delete it.